src/components/calcite-alert/calcite-alert.tsx

```diff
@@ -51,44 +51,32 @@ export class CalciteAlert {
   @Prop() autoDismiss = false;
 
   /** Duration of autoDismiss (only used with `autoDismiss`) */
-  @Prop({ reflect: true, mutable: true }) autoDismissDuration: "fast" | "medium" | "slow" = this
-    .autoDismiss
+  @Prop({ reflect: true }) autoDismissDuration: "fast" | "medium" | "slow" = this.autoDismiss
```

Comment:
It might help to have a stateful prop `duration` to handle this. `duration` could depend on `autoDismissDuration` and `autoDismiss` to calculate what to do.

Reply:
May apply to other places

Reply:
Nice suggestion, but out of scope. Can you create an issue for this?

Reply:
https://github.com/Esri/calcite-components/issues/1002

Reply:
Thanks!